core/templates/services/svg-sanitizer.service.ts

```diff
@@ -71,21 +71,33 @@ export class SvgSanitizerService {
       // we need to replace these decimals with a letter so that it's easier
       // to process and validate the filenames.
       if (node.tagName.toLowerCase() === 'svg') {
-        dimensions.height = (
-          (node.getAttribute('height').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
-        dimensions.width = (
-          (node.getAttribute('width').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
+        let attrHeight = node.getAttribute('height');
+        if (attrHeight) {
+          let height = attrHeight.match(/\d+\.*\d*/g);
+          if (height) {
+            dimensions.height = height[0].replace('.', 'd');
+          }
+        }
+        let attrWidth = node.getAttribute('width');
+        if (attrWidth) {
+          let width = attrWidth.match(/\d+\.*\d*/g);
+          if (width) {
+            dimensions.width = width[0].replace('.', 'd');
+          }
+        }
         // This attribute is useful for the vertical alignment of the
         // Math SVG while displaying inline with other text.
         // Math SVGs don't necessarily have a vertical alignment, in that
         // case we assign it zero.
-        let styleValue = node.getAttribute('style').match(/\d+\.*\d*/g);
-        if (styleValue) {
-          dimensions.verticalPadding = styleValue[0].replace('.', 'd');
-        } else {
-          dimensions.verticalPadding = '0';
+        let styleValue: RegExpMatchArray | null;
+        let attrStyle = node.getAttribute('style');
+        if (attrStyle) {
```

Comment:
ditto, what happens if `attrStyle` does not exist?

Reply:
Deafult value will be used for dimensions.verticalPadding, we also have unit tests covering these cases